Issue #creation-library : feat : slider and translation added

pradeepkumarcm96 opened the pull request.

kiranharidas187 commented on projects/collection-editor-library/src/lib/components/slider/slider.component.html:
```diff
@@ -0,0 +1,17 @@
+    <label class="fs-0-92 font-weight-bold">Slider Value<span class="sb-required">*</span></label>
+    <div class="form-container flex-container">
+        <div>
+            <label><strong>Left</strong></label>
```
add in language pack

kiranharidas187 commented on projects/collection-editor-library/src/lib/components/slider/slider.component.html:
```diff
@@ -0,0 +1,17 @@
+            <input class="textbox" placeholder="Add minimum value" type="number">
+        </div>
+        <div>
+            <label><strong>Step size</strong></label>
```
add in language pack

kiranharidas187 commented on projects/collection-editor-library/src/lib/components/slider/slider.component.html:
```diff
@@ -0,0 +1,17 @@
+            <input class="textbox" placeholder="Add step size" type="number">
+        </div>
+        <div>
+            <label><strong>Right</strong></label>
```
add in language pack

bharatkashyap committed:
wip: single-sourcing (use questionSet as target)

bharatkashyap committed:
feat: questionSet as target, objectType: question

bharatkashyap committed:
wip: add review, publish, reject for question

santoshilimi committed:
#SB-26662 chore: updated package version

santoshilimi committed:
#SB-26662 feat: Increase the content size limit to 150 MB

bharatkashyap committed:
feat: reviewer screen for question object

pallakartheekreddy committed:
Merge pull request #366 from santoshilimi/4.4.0

#SB-26662 feat: Increase the content size limit to 150 MB

bharatkashyap committed:
feat: interpolate publish title

bharatkashyap committed:
wip: sourcing approve and reject

pradeepkumarcm96 committed:
Issue #collection-editor : test :test case added

venkateshwarans committed:
Issue #SB-21912 fix: branching logic test cases

Merge pull request #367 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Merge branch 'release-4.4.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-000 chore: merge release 4.4 into sl-survey

Merge pull request #368 from venkateshwarans/sl-survey

bharatkashyap committed:
feat: sourcing approve/reject questions

bharatkashyap committed:
feat: emit collection data, honour `editable`

mangeshmane committed:
Issue #SB-23256 fix: Question set section form min height

Commit:
Merge pull request #369 from mangeshmane/local-4.4.0

Issue #SB-23256 fix: Question set section form min height

mangeshmane committed:
Issue #SB-23256 fix: Question set section form min height

mangeshmane committed:
Merge remote-tracking branch 'upstream/release-4.4.0' into local-4.4.0

pallakartheekreddy committed:
Merge pull request #370 from mangeshmane/local-4.4.0

Issue #SB-23256 fix: Update the sbform and collection editor package version

pradeepkumarcm96 committed:
Issue #collection-editor : dependent question show option hide and show fix

bharatkashyap committed:
fix: remove index for question objectType

bharatkashyap committed:
fix: move getFrameworkDetails back

venkateshwarans committed:
Issue #SB-21820 fix: dependent question branching logic

venkateshwarans committed:
Issue #SB-21912 fix: branchingLogic and dependent question show option hide

Merge pull request #371 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/components/question/question.component.ts

venkateshwarans committed:
Merge branch 'release-4.4.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-21820 fix: New question type creation

venkateshwarans committed:
Issue #SB-21820 fix: New question type creation

mangeshmane committed:
Issue #SB-23256 fix: Update the sbform package version

mangeshmane committed:
Issue #23256 fix: Update the package version

pallakartheekreddy committed:
Merge pull request #373 from mangeshmane/local-4.4.0

Issue #SB-23256 fix: Update the sbform package version

bharatkashyap committed:
fix: `ng-template` with `ngIf` rendering issue

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

itsvick committed:
Issue #SB-27298 fix: Updated QUML version

itsvick committed:
Issue #SB-27298 fix: Updated package version

bharatkashyap committed:
fix: sourcing update questions

venkateshwarans committed:
Merge branch 'release-4.4.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-21912 fix: Dependent Questions restore the previsouly selected node

itsvick committed:
Issue #SB-0000 fix: Updated QUML player

pallakartheekreddy committed:
Merge pull request #374 from itsvick/release-4.4.0

Issue #SB-27298 fix: Updated QUML version

santoshilimi committed:
#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

santoshilimi committed:
#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

santoshilimi committed:
#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

rajnishdargan committed:
Issue #SB-0000 chore: updating sb-form version

itsvick committed:
Issue #SB-27505 fix: Updated package version for quml player

itsvick committed:
Issue #SB-27505 fix: Updated package version

pallakartheekreddy committed:
Merge pull request #376 from rajnishdargan/release-4.4.0

Issue #SB-0000 chore: updating sb-form version

itsvick committed:
Issue #SB-27505 fix: Updated package version

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

pallakartheekreddy committed:
Merge pull request #377 from itsvick/release-4.4.0

Issue #SB-27505 fix: Updated package version for quml player

rajnishdargan committed:
Issue #SB-0000 chore: updating sb-form version

pallakartheekreddy committed:
Merge pull request #378 from rajnishdargan/release-4.4.0

Issue #SB-0000 chore: updating sb-form version

santoshilimi committed:
#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

santoshilimi committed:
#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

santoshilimi committed:
#SB-27183 feat: fixed test cases

santoshilimi committed:
#SB-27183 feat: fixed test cases

santoshilimi committed:
#SB-27183 feat: updated npm version

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

itsvick committed:
Issue #SB-0000 feat: Updated Video player version

pallakartheekreddy committed:
Merge pull request #379 from itsvick/release-4.4.0

Issue #SB-0000 feat: Updated Video player version

santoshilimi committed:
#SB-27183 feat: updated test cases

santoshilimi committed:
#SB-27183 feat: updated test cases

santoshilimi committed:
#SB-27183 feat: updated npm version

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

itsvick committed:
Issue #SB-0000 feat: Updated Video player version

pallakartheekreddy committed:
Merge pull request #380 from itsvick/release-4.4.0

Issue #SB-0000 feat: Updated Video player version

itsvick committed:
Merge branch 'release-4.4.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.4.0

itsvick committed:
Issue #SB-27591 fix: Updated QUML version

itsvick committed:
Issue #SB-27591 fix: Updated QUML version

pallakartheekreddy committed:
Merge pull request #381 from itsvick/release-4.4.0

Issue #SB-27591 fix: Updated QUML version

pallakartheekreddy committed:
Merge pull request #375 from santoshilimi/4.5.0

#SB-27183 feat: Enable linking of same asset to multiple folders in a collection

venkateshwarans committed:
Issue #SB-000 merge: Merge branch 'release-4.5.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/services/config/label.config.json

Commit:
Merge branch 'release-4.5.0' into release-4.4.0

Commit:
Merge pull request #382 from Sunbird-Ed/release-4.4.0

Release 4.4.0

mangeshmane committed:
Issue #SB-27613 fix : Hover background color desing changes

mangeshmane committed:
Issue #SB-27613 fix : Hover background color desing changes

mangeshmane committed:
Issue #SB-26446 style: Enable transcripts for video content screen desing changes

Commit:
Merge pull request #383 from mangeshmane/4.5-local

Issue #SB-27613 fix : Hover background color design changes

pallakartheekreddy committed:
Issue #SB-26853 fix: added sample csv

pallakartheekreddy committed:
Issue #SB-26853 fix: added sample csv

vaibhavbhuva committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

vaibhavbhuva committed:
Merge branch 'release-4.5.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into release-4.5.0

* 'release-4.5.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor: (25 commits)
  Issue #SB-26853 fix: added sample csv
  Issue #SB-26853 fix: added sample csv
  Issue #SB-26446 style: Enable transcripts for video content screen desing changes
  Issue #SB-27613 fix : Hover background color desing changes
  Issue #SB-27613 fix : Hover background color desing changes
  Issue #SB-27591 fix: Updated QUML version
  Issue #SB-27591 fix: Updated QUML version
  Issue #SB-0000 feat: Updated Video player version
  Issue #SB-0000 feat: Updated Video player version
  Issue #SB-0000 chore: updating sb-form version
  Issue #SB-27505 fix: Updated package version
  Issue #SB-27505 fix: Updated package version
  ...

vaibhavbhuva committed:
Issue @SB-0000 fix: package version updated

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

santoshilimi committed:
Issue #SB-26853 feat: Test case

santoshilimi committed:
Issue #SB-26853 feat: Test case

santoshilimi committed:
Issue #SB-26853 feat: Test case

santoshilimi committed:
Issue #SB-26853 feat: Test case

manojSRawat committed:
feat: enable submit for review while question create

mangeshmane committed:
Issue #SB-27613 fix : Hover background color design changes

pallakartheekreddy committed:
Merge pull request #386 from mangeshmane/4.5-local

Issue #SB-27613 fix :  Already present in this folder label background hover background color

manojSRawat committed:
fix: mcq default 4 option

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

pallakartheekreddy committed:
Merge pull request #385 from santoshilimi/26853

Issue #SB-26853 feat: Bulk upload questions (MCQ)

vaibhavbhuva committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

itsvick committed:
SB-27110 fix: Updated quml version

vaibhavbhuva committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ

vaibhavbhuva committed:
Issue #SB-0000 chore : package version updated

vaibhavbhuva committed:
Issue #SB-0000 chore : package version updated

pallakartheekreddy committed:
Merge pull request #389 from vaibhavbhuva/SB-26853

Issue #SB-26853 feat: Bulk upload questions (MCQ)

itsvick committed:
Issue #SB-27764 fix: Updated quml player

santoshilimi committed:
Issue #SB-26853 feat: Bulk upload questions (MCQ)

pallakartheekreddy committed:
Merge pull request #390 from santoshilimi/26853

Issue #SB-26853 feat: Bulk upload questions (MCQ)

pallakartheekreddy committed:
Merge pull request #388 from itsvick/release-4.5.0

SB-27110 fix: Updated quml version

vaibhavbhuva committed:
Issue #SB-0000 chore : package version updated

vaibhavbhuva committed:
Merge branch 'SB-26853' of https://github.com/vaibhavbhuva/sunbird-collection-editor-1 into release-4.5.0

* 'SB-26853' of https://github.com/vaibhavbhuva/sunbird-collection-editor-1:
  Issue #SB-0000 chore : package version updated
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-0000 chore : package version updated
  Issue #SB-0000 chore : package version updated
  Issue #SB-26853 feat: Bulk upload questions (MCQ
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue #SB-26853 feat: Test case
  Issue #SB-26853 feat: Test case
  Issue #SB-26853 feat: Test case
  Issue #SB-26853 feat: Test case
  Issue #SB-26853 feat: Bulk upload questions (MCQ)
  Issue @SB-0000 fix: package version updated
  Issue #SB-26853 feat: Bulk upload questions (MCQ)

pallakartheekreddy committed:
Merge pull request #391 from vaibhavbhuva/release-4.5.0

Issue #SB-26853 feat: Bulk upload questions (MCQ)

vaibhavbhuva committed:
Issue #SB-26853 fix : Bulk question job APIs issue fixes

pallakartheekreddy committed:
Merge pull request #392 from vaibhavbhuva/release-4.5.0

Issue #SB-26853 fix : Bulk question job APIs issue fixes

mangeshmane committed:
Issue #SB-26853 fix: Bulk upload popup loader issue

mangeshmane committed:
Issue #SB-26853 fix: Bulk upload popup loader issue

pallakartheekreddy committed:
Merge pull request #393 from mangeshmane/4.5-local

Issue #SB-26853 fix: Bulk upload popup loader issue

itsvick committed:
Merge branch 'release-4.5.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.5.0

itsvick committed:
Issue #SB-26590 fix: Updated quml player

itsvick committed:
Issue #SB-26590 fix: Updated version

pallakartheekreddy committed:
Merge pull request #394 from itsvick/release-4.5.0

Issue #SB-26590 fix: Updated quml player

venkateshwarans committed:
Issue #SB-000 merge: Merge branch 'release-4.5.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/collection-editor-library.module.ts
#	projects/collection-editor-library/src/lib/services/config/label.config.json
#	src/app/data.ts

santoshilimi committed:
#SB-26853 feat: Bulk upload questions (MCQ) in prescribed format - Part 2

Commit:
Merge pull request #395 from santoshilimi/4.5.1

#SB-26853 feat: Bulk upload questions (MCQ) in prescribed format - Part 2

santoshilimi committed:
#SB-27490 fix: Root node itself is linked to the root node hierarchy

santoshilimi committed:
#SB-27490 fix: Root node itself is linked to the root node hierarchy

Commit:
Merge pull request #396 from santoshilimi/27490

#SB-27490 fix: Root node itself is linked to the root node hierarchy

venkateshwarans committed:
Issue #SB-21820 feat: dependent question branching logic

venkateshwarans committed:
Issue #SB-21820 fix: New question type creation

Merge pull request #372 from venkateshwarans/sl-survey

dhruvarya9654 committed:
approve reject questions

dhruvarya9654 committed:
send back to correction fix

itsvick committed:
Issue #SB-27864 fix: Updated quml player version

itsvick committed:
Merge branch 'release-4.5.0' of github.com:Sunbird-Ed/sunbird-collection-editor into release-4.5.0

itsvick committed:
Issue #SB-27864 fix: Updated quml player version

pallakartheekreddy committed:
Merge pull request #397 from itsvick/release-4.5.0

Issue #SB-27864 fix: Updated quml player version

dhruvarya9654 committed:
save as draft fix

pradeepkumarcm96 committed:
Issue #collection-Editor : feat : branching logic update

venkateshwarans committed:
Issue #SB-21820 fix: branching logic update

Merge pull request #398 from shikshalokam/collection-editor-Pk

pradeepkumarcm96 committed:
Issue #collection-editor : fix : top dropdown for child

vaibhavbhuva committed:
Issue #SB-0000 fix: package version updated

vaibhavbhuva committed:
Issue #SB-27351 feat : Sourcing: Updating latest players in sourcing portal & editor

vaibhavbhuva committed:
Issue #SB-0000 fix: package version updated

pallakartheekreddy committed:
Merge pull request #404 from vaibhavbhuva/release-4.6.0

Issue #SB-27351 feat : Sourcing: Updating latest players in sourcing portal & editor

pradeepkumarcm96 committed:
Issue #collection-editor : fix : branching logic update

venkateshwarans committed:
Issue #SB-21820 fix: top dropdown for child

Merge pull request #402 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Merge branch 'release-4.6.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-000 chore: merging release-4.6 to sl-survey

Merge pull request #405 from venkateshwarans/sl-survey

pradeepkumarcm96 committed:
Issue #collection-editor : fix in child question and parent

pradeepkumarcm96 committed:
issue #collection-editor : fixes

venkateshwarans committed:
Issue #SB-21820 fix: branching logic fix in child question and parent

Merge pull request #406 from shikshalokam/collection-editor-Pk

manojSRawat committed:
feat: question reject key chagne

pradeepkumarcm96 committed:
Issue #collection-editor : dependent question issue fix

venkateshwarans committed:
Issue #SB-21820 feat: dependent question branching logic

venkateshwarans committed:
Issue #SB-21820 fix: dependent question issue fix

Merge pull request #407 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

pradeepkumarcm96 committed:
Issue #collection-editor : hierarchy update

venkateshwarans committed:
Issue #SB-21820 fix: branching logic fix in child question and parent hierarchy update

Merge pull request #408 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-21820 feat: dependent question branching logic

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop

venkateshwarans committed:
Issue #SB-21820 fix: dependent question branching logic

Merge pull request #409 from venkateshwarans/sl-survey

venkateshwarans committed:
Issue #SB-21820 fix: dependent question indendation

venkateshwarans committed:
Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

venkateshwarans committed:
Issue #SB-21820 fix: dependent question indentation

Merge pull request #410 from venkateshwarans/sl-survey

pradeepkumarcm96 committed:
Issue #collection-editor : drag and drop fix

pradeepkumarcm96 committed:
Merge remote-tracking branch 'upstream/sl-survey' into collection-editor-Pk

NirmalBohra1 committed:
test cases issues fixed

dhruvarya9654 committed:
create question form - maxScore type fix

dhruvarya9654 committed:
Merge branch 'refactor-4.6' of https://github.com/manojSRawat/sunbird-collection-editor-1 into refactor-4.6

NirmalBohra1 committed:
test case

pradeepkumarcm96 committed:
Issue #collection-editor : branching logic update

pradeepkumarcm96 committed:
Issue #collection-editor : branching logic update

dhruvarya9654 committed:
question-component testcases

pradeepkumarcm96 committed:
Issue #collection-editor : comments resolved

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

Merge pull request #411 from shikshalokam/collection-editor-Pk

dhruvarya9654 committed:
testcases for question, header, fancy-tree components

manojSRawat committed:
chore: package name restore

dhruvarya9654 committed:
editor flow changes for MTF, and other categories

dhruvarya9654 committed:
Merge branch 'refactor-4.6' of https://github.com/manojSRawat/sunbird-collection-editor-1 into refactor-4.6

venkateshwarans committed:
Issue #SB-21820 fix: dependent question indendation

venkateshwarans committed:
Issue #SB-21820 fix: dependent question indentation

Merge pull request #412 from venkateshwarans/sl-survey

pradeepkumarcm96 committed:
Issue #collection-editor : fix : branchinglogic in tree structure

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

Merge pull request #413 from shikshalokam/collection-editor-Pk

manojSRawat committed:
feat: question create sb form fix

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

Merge pull request #414 from venkateshwarans/sl-survey

dhruvarya9654 committed:
editor-component testcases

pradeepkumarcm96 committed:
Issue #collection-editor : test : test case added

venkateshwarans committed:
Issue #SB-21820 fix: dependent question test case added

Merge pull request #415 from shikshalokam/collection-editor-Pk

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

venkateshwarans committed:
Issue #SB-21820 merge: Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

venkateshwarans committed:
Issue #SB-21820 fix: dependent question drag and drop fix

Merge pull request #417 from venkateshwarans/sl-survey

dhruvarya9654 committed:
reviewed changes

manojSRawat committed:
merge release 4.6.0

santoshilimi committed:
#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

santoshilimi committed:
#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

santoshilimi committed:
#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

santoshilimi committed:
#SB-25591 fix: updated npm package version

dhruvarya9654 committed:
MCQ numberOfOptions config

dhruvarya9654 committed:
Merge branch 'refactor-4.6' of https://github.com/manojSRawat/sunbird-collection-editor-1 into refactor-4.6

dhruvarya9654 committed:
MCQ numberOfOptions config

dhruvarya9654 committed:
objectType for header component

pallakartheekreddy committed:
Merge pull request #419 from santoshilimi/asset

#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

dhruvarya9654 committed:
Merge branch 'release-4.6.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into refactor-4.6

santoshilimi committed:
#SB-25591 fix: updated npm package version

santoshilimi committed:
#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

santoshilimi committed:
Merge remote-tracking branch 'upstream/release-4.6.0' into asset

pallakartheekreddy committed:
Merge pull request #420 from santoshilimi/asset

#SB-25591 fix: Sourcing: Integration of Asset V4 APIs

dhruvarya9654 committed:
editor component testcases

dhruvarya9654 committed:
editor component testcases

dhruvarya9654 committed:
meta-form testcases

dhruvarya9654 committed:
meta-form testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

dhruvarya9654 committed:
added testcases

manojSRawat committed:
Fancy tree: test cases added

manojSRawat committed:
fancy tree: revert test case

manojSRawat committed:
Sanitize pipe test case added

manojSRawat committed:
chore: version update

manojSRawat committed:
package version conflict resolve

manojSRawat committed:
chore: version update

pallakartheekreddy committed:
Merge pull request #399 from manojSRawat/refactor-4.6

[Draft] Refactor 1.0

manojSRawat committed:
Creator added in question create

venkateshwarans committed:
Issue #SB-21820 fix: adding new test data to sample application

venkateshwarans committed:
Merge branch 'sl-survey' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

manojSRawat committed:
feat: question detail visibiliy issue fix and topic selector UI fix

dhruvarya9654 committed:
sourcing: send back for correction (pending)

dhruvarya9654 committed:
Merge branch 'refactor-fix' of https://github.com/manojSRawat/sunbird-collection-editor-1 into refactor-fix

venkateshwarans committed:
Issue #SB-000 merge: Merge branch 'release-4.6.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/components/editor/editor.component.spec.ts
#	projects/collection-editor-library/src/lib/components/editor/editor.component.ts
#	projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.spec.data.ts
#	projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.spec.ts
#	projects/collection-editor-library/src/lib/components/options/options.component.html
#	projects/collection-editor-library/src/lib/components/options/options.component.ts
#	projects/collection-editor-library/src/lib/components/question/question.component.html
#	projects/collection-editor-library/src/lib/components/question/question.component.spec.data.ts
#	projects/collection-editor-library/src/lib/components/question/question.component.spec.ts
#	projects/collection-editor-library/src/lib/components/question/question.component.ts
#	projects/collection-editor-library/src/lib/services/config/label.config.json
#	projects/collection-editor-library/src/lib/services/editor/editor.service.ts
#	src/app/app.component.html
#	src/app/app.component.ts
#	src/app/data.ts

manojSRawat committed:
chore: version key fix

manojSRawat committed:
chore: version update

manojSRawat committed:
sendback for correction test case remove as API not working

pallakartheekreddy committed:
Merge pull request #421 from manojSRawat/refactor-fix

Refactor fix

venkateshwarans committed:
Issue #SB-000 merge: Merge branch 'release-4.6.0' of https://github.com/Sunbird-Ed/sunbird-collection-editor into sl-survey

# Conflicts:
#	projects/collection-editor-library/src/lib/components/editor/editor.component.html
#	projects/collection-editor-library/src/lib/components/question/question.component.scss
#	projects/collection-editor-library/src/lib/components/question/question.component.spec.ts

venkateshwarans committed:
Issue #SB-000 merge: fixing test case

venkateshwarans committed:
Issue #SB-000 merge: fixing test case

venkateshwarans committed:
Issue #SB-000 merge: fixing test case

venkateshwarans committed:
Issue #SB-000 merge: fixing test case

venkateshwarans committed:
Issue #SB-000 chore: merging release-4.6 to sl-survey

Merge pull request #422 from venkateshwarans/sl-survey

pradeepkumarcm96 committed:
Issue #collection-editor : obs with rubrics

pradeepkumarcm96 committed:
Merge remote-tracking branch 'upstream/sl-survey' into collection-editor-Pk

pradeepkumarcm96 committed:
Issue #collection-editor : test : test case fix
